### PR TITLE
LongHaul - Add operation name to logs

### PR DESCRIPTION
**Pull request by tmahmood-microsoft**

### CODE REVIEW

File: e2e/LongHaul/device/IotHub.cs [hunk 1]
```diff
@@ -99,13 +99,14 @@ public async Task InitializeAsync()
         /// Frequently send telemetry messages to the hub.
         /// </summary>
         /// <param name="ct">The cancellation token</param>
-        public async Task SendTelemetryMessagesAsync(CancellationToken ct)
+        public async Task SendTelemetryMessagesAsync(CancellationToken ct, Logger logger)
```
Comment (drwill-ms) [inline e2e/LongHaul/device/IotHub.cs:102]: swap parameter order. CTs always come last in .NET, by convention.

Comment (drwill-ms) [inline e2e/LongHaul/device/IotHub.cs:102]: static?

File: e2e/LongHaul/service/DeviceOperations.cs [hunk 2]
```diff
@@ -61,43 +62,44 @@ public async Task InvokeDirectMethodAsync(CancellationToken ct)
...
                 }
 
                 await Task.Delay(s_directMethodInvokeInterval, ct).ConfigureAwait(false);
             }
         }
 
-        public async Task SetDesiredPropertiesAsync(string keyName, string properties, CancellationToken ct)
+        public async Task SetDesiredPropertiesAsync(string keyName, string properties, CancellationToken ct, Logger logger)
         {
+            logger.LoggerContext.Add("OperationName", "SetDesiredProperties");
```
Comment (drwill-ms) [inline e2e/LongHaul/service/DeviceOperations.cs:83]: Make these into constants somewhere

**Review by drwill-ms — APPROVED** [review]

**Review by patilsnr — APPROVED** [review]
In the places where `_logger` becomes `logger`, is that because we have to make it non-private?

**Pull request merged**